docs/contracts.rst

```diff
@@ -733,6 +732,61 @@ For example:
        >>> rich_logs[0]['args']
        {'myArg': 12345}
 
+   By default, an error will be raised if the log doesn't match the provided ABI.
+   However, errors can be optionally ignored by passing in the ``ignore`` or
+   ``discard`` flag located in the ``EventLogErrorFlags`` class. The ``discard``
+   flag will discard any logs that raise an error during processing, and return
+   the rest. The ``ignore`` flag will return the raw log that caused the error,
+   without any processing, and will process any logs that do not raise an error.
+   An ``EventLogErrorFlag`` is imported from web3/_utils/events.py.
+
+   .. code-block:: python
+
+       >>> from web3._utils.events import EventLogErrorFlags
```

Comment:
I'm thinking that the `EventLogErrorFlags` should be private and we only expose the individual flags for import.  Also, lets make them capcase since I think that lines up better with APIs elsewhere in the standard library. `DISCARD`, `IGNORE`

Reply:
That sounds fine to me. I couldn't decide 😆 I left them as lowercase for the future so that if we do end up defining them as methods, we wouldn't have `def STRICT`, but I think uppercase makes more sense for now 👍